portugues/colabora/follow/follow.md

```diff
@@ -0,0 +1 @@
+Parabéns, agora você pode receber todas as atualizações deste desenvolvedor que fez um código com bug. Será uma boa ideia?
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.